### PR TITLE
adding installer jobs for 4.6 4.7 s390x ppc64le

### PR DESCRIPTION
**Pull request by LakshmiRavichandran1**

### CODE REVIEW

File: core-services/release-controller/_releases/release-ocp-4.6-ppc64le.json [hunk 1]
```diff
@@ -13,5 +13,9 @@
     "tag":{"tagRef":{"name":"4.6"}}
   },
   "verify":{
+    "ppc64le":{
+      "optional":true,
+      "prowJob":{"name":"periodic-ci-openshift-multiarch-master-nightly-4.6-ocp-installer-remote-libvirt-ppc64le"}
```
Comment (deepsm007) [inline core-services/release-controller/_releases/release-ocp-4.6-ppc64le.json:18]: All the jobs name were updated 
`pull-ci-openshift-multiarch-master-nightly-4.6-ocp-installer-remote-libvirt-ppc64le`

Comment (LakshmiRavichandran1) [reply core-services/release-controller/_releases/release-ocp-4.6-ppc64le.json:18]: changed them to use periodic-***

**Review by deepsm007 — APPROVED** [review]

**Review by bradmwilliams — APPROVED** [review]

**Pull request merged**